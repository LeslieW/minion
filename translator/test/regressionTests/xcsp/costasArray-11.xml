<?xml version="1.0" encoding="UTF-8"?>

<instance>
<presentation  maxConstraintArity="11" format="XCSP 2.1" type="CSP"/>

  <domains nbDomains="2">
    <domain name="D0" nbValues="11">1..11</domain>
    <domain name="D1" nbValues="20">-10..-1 1..10</domain>
  </domains>

  <variables nbVariables="63">
    <variable name="V0" domain="D0"/>
    <variable name="V1" domain="D0"/>
    <variable name="V2" domain="D0"/>
    <variable name="V3" domain="D0"/>
    <variable name="V4" domain="D0"/>
    <variable name="V5" domain="D0"/>
    <variable name="V6" domain="D0"/>
    <variable name="V7" domain="D0"/>
    <variable name="V8" domain="D0"/>
    <variable name="V9" domain="D0"/>
    <variable name="V10" domain="D0"/>
    <variable name="V11" domain="D1"/>
    <variable name="V12" domain="D1"/>
    <variable name="V13" domain="D1"/>
    <variable name="V14" domain="D1"/>
    <variable name="V15" domain="D1"/>
    <variable name="V16" domain="D1"/>
    <variable name="V17" domain="D1"/>
    <variable name="V18" domain="D1"/>
    <variable name="V19" domain="D1"/>
    <variable name="V20" domain="D1"/>
    <variable name="V21" domain="D1"/>
    <variable name="V22" domain="D1"/>
    <variable name="V23" domain="D1"/>
    <variable name="V24" domain="D1"/>
    <variable name="V25" domain="D1"/>
    <variable name="V26" domain="D1"/>
    <variable name="V27" domain="D1"/>
    <variable name="V28" domain="D1"/>
    <variable name="V29" domain="D1"/>
    <variable name="V30" domain="D1"/>
    <variable name="V31" domain="D1"/>
    <variable name="V32" domain="D1"/>
    <variable name="V33" domain="D1"/>
    <variable name="V34" domain="D1"/>
    <variable name="V35" domain="D1"/>
    <variable name="V36" domain="D1"/>
    <variable name="V37" domain="D1"/>
    <variable name="V38" domain="D1"/>
    <variable name="V39" domain="D1"/>
    <variable name="V40" domain="D1"/>
    <variable name="V41" domain="D1"/>
    <variable name="V42" domain="D1"/>
    <variable name="V43" domain="D1"/>
    <variable name="V44" domain="D1"/>
    <variable name="V45" domain="D1"/>
    <variable name="V46" domain="D1"/>
    <variable name="V47" domain="D1"/>
    <variable name="V48" domain="D1"/>
    <variable name="V49" domain="D1"/>
    <variable name="V50" domain="D1"/>
    <variable name="V51" domain="D1"/>
    <variable name="V52" domain="D1"/>
    <variable name="V53" domain="D1"/>
    <variable name="V54" domain="D1"/>
    <variable name="V55" domain="D1"/>
    <variable name="V56" domain="D1"/>
    <variable name="V57" domain="D1"/>
    <variable name="V58" domain="D1"/>
    <variable name="V59" domain="D1"/>
    <variable name="V60" domain="D1"/>
    <variable name="V61" domain="D1"/>
    <variable name="V62" domain="D1"/>
  </variables>

  <predicates nbPredicates="2">
    <predicate name="P0">
      <parameters> int X0 int X1 int X2 int X3</parameters>
      <expression> 
        <functional>ne(sub(X0,X1),sub(X2,X3))</functional>
      </expression>
    </predicate>
    <predicate name="P1">
      <parameters> int X0 int X1 int X2</parameters>
      <expression> 
        <functional>eq(sub(X0,X1),X2)</functional>
      </expression>
    </predicate>
  </predicates>

  <constraints nbConstraints="62">
    <constraint name="C0" arity="11" scope="V0 V1 V2 V3 V4 V5 V6 V7 V8 V9 V10" reference="global:allDifferent"/>
    <constraint name="C1" arity="3" scope="V0 V1 V11" reference="P1">
      <parameters>V0 V1 V11</parameters> 
    </constraint>
    <constraint name="C2" arity="3" scope="V1 V2 V12" reference="P1">
      <parameters>V1 V2 V12</parameters> 
    </constraint>
    <constraint name="C3" arity="3" scope="V2 V3 V13" reference="P1">
      <parameters>V2 V3 V13</parameters> 
    </constraint>
    <constraint name="C4" arity="3" scope="V3 V4 V14" reference="P1">
      <parameters>V3 V4 V14</parameters> 
    </constraint>
    <constraint name="C5" arity="3" scope="V4 V5 V15" reference="P1">
      <parameters>V4 V5 V15</parameters> 
    </constraint>
    <constraint name="C6" arity="3" scope="V5 V6 V16" reference="P1">
      <parameters>V5 V6 V16</parameters> 
    </constraint>
    <constraint name="C7" arity="3" scope="V6 V7 V17" reference="P1">
      <parameters>V6 V7 V17</parameters> 
    </constraint>
    <constraint name="C8" arity="3" scope="V7 V8 V18" reference="P1">
      <parameters>V7 V8 V18</parameters> 
    </constraint>
    <constraint name="C9" arity="3" scope="V8 V9 V19" reference="P1">
      <parameters>V8 V9 V19</parameters> 
    </constraint>
    <constraint name="C10" arity="3" scope="V9 V10 V20" reference="P1">
      <parameters>V9 V10 V20</parameters> 
    </constraint>
    <constraint name="C11" arity="10" scope="V11 V12 V13 V14 V15 V16 V17 V18 V19 V20" reference="global:allDifferent"/>
    <constraint name="C12" arity="3" scope="V0 V2 V21" reference="P1">
      <parameters>V0 V2 V21</parameters> 
    </constraint>
    <constraint name="C13" arity="3" scope="V1 V3 V22" reference="P1">
      <parameters>V1 V3 V22</parameters> 
    </constraint>
    <constraint name="C14" arity="3" scope="V2 V4 V23" reference="P1">
      <parameters>V2 V4 V23</parameters> 
    </constraint>
    <constraint name="C15" arity="3" scope="V3 V5 V24" reference="P1">
      <parameters>V3 V5 V24</parameters> 
    </constraint>
    <constraint name="C16" arity="3" scope="V4 V6 V25" reference="P1">
      <parameters>V4 V6 V25</parameters> 
    </constraint>
    <constraint name="C17" arity="3" scope="V5 V7 V26" reference="P1">
      <parameters>V5 V7 V26</parameters> 
    </constraint>
    <constraint name="C18" arity="3" scope="V6 V8 V27" reference="P1">
      <parameters>V6 V8 V27</parameters> 
    </constraint>
    <constraint name="C19" arity="3" scope="V7 V9 V28" reference="P1">
      <parameters>V7 V9 V28</parameters> 
    </constraint>
    <constraint name="C20" arity="3" scope="V8 V10 V29" reference="P1">
      <parameters>V8 V10 V29</parameters> 
    </constraint>
    <constraint name="C21" arity="9" scope="V21 V22 V23 V24 V25 V26 V27 V28 V29" reference="global:allDifferent"/>
    <constraint name="C22" arity="3" scope="V0 V3 V30" reference="P1">
      <parameters>V0 V3 V30</parameters> 
    </constraint>
    <constraint name="C23" arity="3" scope="V1 V4 V31" reference="P1">
      <parameters>V1 V4 V31</parameters> 
    </constraint>
    <constraint name="C24" arity="3" scope="V2 V5 V32" reference="P1">
      <parameters>V2 V5 V32</parameters> 
    </constraint>
    <constraint name="C25" arity="3" scope="V3 V6 V33" reference="P1">
      <parameters>V3 V6 V33</parameters> 
    </constraint>
    <constraint name="C26" arity="3" scope="V4 V7 V34" reference="P1">
      <parameters>V4 V7 V34</parameters> 
    </constraint>
    <constraint name="C27" arity="3" scope="V5 V8 V35" reference="P1">
      <parameters>V5 V8 V35</parameters> 
    </constraint>
    <constraint name="C28" arity="3" scope="V6 V9 V36" reference="P1">
      <parameters>V6 V9 V36</parameters> 
    </constraint>
    <constraint name="C29" arity="3" scope="V7 V10 V37" reference="P1">
      <parameters>V7 V10 V37</parameters> 
    </constraint>
    <constraint name="C30" arity="8" scope="V30 V31 V32 V33 V34 V35 V36 V37" reference="global:allDifferent"/>
    <constraint name="C31" arity="3" scope="V0 V4 V38" reference="P1">
      <parameters>V0 V4 V38</parameters> 
    </constraint>
    <constraint name="C32" arity="3" scope="V1 V5 V39" reference="P1">
      <parameters>V1 V5 V39</parameters> 
    </constraint>
    <constraint name="C33" arity="3" scope="V2 V6 V40" reference="P1">
      <parameters>V2 V6 V40</parameters> 
    </constraint>
    <constraint name="C34" arity="3" scope="V3 V7 V41" reference="P1">
      <parameters>V3 V7 V41</parameters> 
    </constraint>
    <constraint name="C35" arity="3" scope="V4 V8 V42" reference="P1">
      <parameters>V4 V8 V42</parameters> 
    </constraint>
    <constraint name="C36" arity="3" scope="V5 V9 V43" reference="P1">
      <parameters>V5 V9 V43</parameters> 
    </constraint>
    <constraint name="C37" arity="3" scope="V6 V10 V44" reference="P1">
      <parameters>V6 V10 V44</parameters> 
    </constraint>
    <constraint name="C38" arity="7" scope="V38 V39 V40 V41 V42 V43 V44" reference="global:allDifferent"/>
    <constraint name="C39" arity="3" scope="V0 V5 V45" reference="P1">
      <parameters>V0 V5 V45</parameters> 
    </constraint>
    <constraint name="C40" arity="3" scope="V1 V6 V46" reference="P1">
      <parameters>V1 V6 V46</parameters> 
    </constraint>
    <constraint name="C41" arity="3" scope="V2 V7 V47" reference="P1">
      <parameters>V2 V7 V47</parameters> 
    </constraint>
    <constraint name="C42" arity="3" scope="V3 V8 V48" reference="P1">
      <parameters>V3 V8 V48</parameters> 
    </constraint>
    <constraint name="C43" arity="3" scope="V4 V9 V49" reference="P1">
      <parameters>V4 V9 V49</parameters> 
    </constraint>
    <constraint name="C44" arity="3" scope="V5 V10 V50" reference="P1">
      <parameters>V5 V10 V50</parameters> 
    </constraint>
    <constraint name="C45" arity="6" scope="V45 V46 V47 V48 V49 V50" reference="global:allDifferent"/>
    <constraint name="C46" arity="3" scope="V0 V6 V51" reference="P1">
      <parameters>V0 V6 V51</parameters> 
    </constraint>
    <constraint name="C47" arity="3" scope="V1 V7 V52" reference="P1">
      <parameters>V1 V7 V52</parameters> 
    </constraint>
    <constraint name="C48" arity="3" scope="V2 V8 V53" reference="P1">
      <parameters>V2 V8 V53</parameters> 
    </constraint>
    <constraint name="C49" arity="3" scope="V3 V9 V54" reference="P1">
      <parameters>V3 V9 V54</parameters> 
    </constraint>
    <constraint name="C50" arity="3" scope="V4 V10 V55" reference="P1">
      <parameters>V4 V10 V55</parameters> 
    </constraint>
    <constraint name="C51" arity="5" scope="V51 V52 V53 V54 V55" reference="global:allDifferent"/>
    <constraint name="C52" arity="3" scope="V0 V7 V56" reference="P1">
      <parameters>V0 V7 V56</parameters> 
    </constraint>
    <constraint name="C53" arity="3" scope="V1 V8 V57" reference="P1">
      <parameters>V1 V8 V57</parameters> 
    </constraint>
    <constraint name="C54" arity="3" scope="V2 V9 V58" reference="P1">
      <parameters>V2 V9 V58</parameters> 
    </constraint>
    <constraint name="C55" arity="3" scope="V3 V10 V59" reference="P1">
      <parameters>V3 V10 V59</parameters> 
    </constraint>
    <constraint name="C56" arity="4" scope="V56 V57 V58 V59" reference="global:allDifferent"/>
    <constraint name="C57" arity="3" scope="V0 V8 V60" reference="P1">
      <parameters>V0 V8 V60</parameters> 
    </constraint>
    <constraint name="C58" arity="3" scope="V1 V9 V61" reference="P1">
      <parameters>V1 V9 V61</parameters> 
    </constraint>
    <constraint name="C59" arity="3" scope="V2 V10 V62" reference="P1">
      <parameters>V2 V10 V62</parameters> 
    </constraint>
    <constraint name="C60" arity="3" scope="V60 V61 V62" reference="global:allDifferent"/>
    <constraint name="C61" arity="4" scope="V0 V1 V9 V10" reference="P0">
      <parameters>V0 V9 V1 V10</parameters> 
    </constraint>
  </constraints>

</instance>
