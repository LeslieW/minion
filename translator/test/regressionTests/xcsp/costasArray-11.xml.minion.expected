MINION 3

#This file has been automatically generated by TAILOR v0.2
#bug-reports: andrea@cs.st-and.ac.uk

# amount of common subexpressions used:0
# amount of inferred common subexpressions used:0
# amount of original variables saved (over direct equality):0
# amount of constraints:118
# amount of auxiliary variables: 2

**VARIABLES**
DISCRETE V0 {1..11}
DISCRETE V1 {1..11}
DISCRETE V2 {1..11}
DISCRETE V3 {1..11}
DISCRETE V4 {1..11}
DISCRETE V5 {1..11}
DISCRETE V6 {1..11}
DISCRETE V7 {1..11}
DISCRETE V8 {1..11}
DISCRETE V9 {1..11}
DISCRETE V10 {1..11}
DISCRETE V11 {-10..10}
DISCRETE V12 {-10..10}
DISCRETE V13 {-10..10}
DISCRETE V14 {-10..10}
DISCRETE V15 {-10..10}
DISCRETE V16 {-10..10}
DISCRETE V17 {-10..10}
DISCRETE V18 {-10..10}
DISCRETE V19 {-10..10}
DISCRETE V20 {-10..10}
DISCRETE V21 {-10..10}
DISCRETE V22 {-10..10}
DISCRETE V23 {-10..10}
DISCRETE V24 {-10..10}
DISCRETE V25 {-10..10}
DISCRETE V26 {-10..10}
DISCRETE V27 {-10..10}
DISCRETE V28 {-10..10}
DISCRETE V29 {-10..10}
DISCRETE V30 {-10..10}
DISCRETE V31 {-10..10}
DISCRETE V32 {-10..10}
DISCRETE V33 {-10..10}
DISCRETE V34 {-10..10}
DISCRETE V35 {-10..10}
DISCRETE V36 {-10..10}
DISCRETE V37 {-10..10}
DISCRETE V38 {-10..10}
DISCRETE V39 {-10..10}
DISCRETE V40 {-10..10}
DISCRETE V41 {-10..10}
DISCRETE V42 {-10..10}
DISCRETE V43 {-10..10}
DISCRETE V44 {-10..10}
DISCRETE V45 {-10..10}
DISCRETE V46 {-10..10}
DISCRETE V47 {-10..10}
DISCRETE V48 {-10..10}
DISCRETE V49 {-10..10}
DISCRETE V50 {-10..10}
DISCRETE V51 {-10..10}
DISCRETE V52 {-10..10}
DISCRETE V53 {-10..10}
DISCRETE V54 {-10..10}
DISCRETE V55 {-10..10}
DISCRETE V56 {-10..10}
DISCRETE V57 {-10..10}
DISCRETE V58 {-10..10}
DISCRETE V59 {-10..10}
DISCRETE V60 {-10..10}
DISCRETE V61 {-10..10}
DISCRETE V62 {-10..10}

# auxiliary variables
DISCRETE aux0 {-10..10}
DISCRETE aux1 {-10..10}

# aliases

**SEARCH**

PRINT [
[V0],[V1],[V2],[V3],[V4],[V5],[V6],[V7],
[V8],[V9],[V10],[V11],[V12],[V13],[V14],[V15],
[V16],[V17],[V18],[V19],[V20],[V21],[V22],[V23],
[V24],[V25],[V26],[V27],[V28],[V29],[V30],[V31],
[V32],[V33],[V34],[V35],[V36],[V37],[V38],[V39],
[V40],[V41],[V42],[V43],[V44],[V45],[V46],[V47],
[V48],[V49],[V50],[V51],[V52],[V53],[V54],[V55],
[V56],[V57],[V58],[V59],[V60],[V61],[V62]]

VARORDER [
V0,V1,V2,V3,V4,V5,V6,V7,
V8,V9,V10,V11,V12,V13,V14,V15,
V16,V17,V18,V19,V20,V21,V22,V23,
V24,V25,V26,V27,V28,V29,V30,V31,
V32,V33,V34,V35,V36,V37,V38,V39,
V40,V41,V42,V43,V44,V45,V46,V47,
V48,V49,V50,V51,V52,V53,V54,V55,
V56,V57,V58,V59,V60,V61,V62,
aux0,aux1]


**CONSTRAINTS**

weightedsumgeq([1,-1], [V0,V9], aux0)
weightedsumleq([1,-1], [V0,V9], aux0)
weightedsumgeq([1,-1], [V1,V10], aux1)
weightedsumleq([1,-1], [V1,V10], aux1)
diseq(aux1, aux0)
alldiff([ V60, V61, V62])
weightedsumgeq([1,-1], [V2,V10], V62)
weightedsumleq([1,-1], [V2,V10], V62)
weightedsumgeq([1,-1], [V1,V9], V61)
weightedsumleq([1,-1], [V1,V9], V61)
weightedsumgeq([1,-1], [V0,V8], V60)
weightedsumleq([1,-1], [V0,V8], V60)
alldiff([ V56, V57, V58, V59])
weightedsumgeq([1,-1], [V3,V10], V59)
weightedsumleq([1,-1], [V3,V10], V59)
weightedsumgeq([1,-1], [V2,V9], V58)
weightedsumleq([1,-1], [V2,V9], V58)
weightedsumgeq([1,-1], [V1,V8], V57)
weightedsumleq([1,-1], [V1,V8], V57)
weightedsumgeq([1,-1], [V0,V7], V56)
weightedsumleq([1,-1], [V0,V7], V56)
alldiff([ V51, V52, V53, V54, V55])
weightedsumgeq([1,-1], [V4,V10], V55)
weightedsumleq([1,-1], [V4,V10], V55)
weightedsumgeq([1,-1], [V3,V9], V54)
weightedsumleq([1,-1], [V3,V9], V54)
weightedsumgeq([1,-1], [V2,V8], V53)
weightedsumleq([1,-1], [V2,V8], V53)
weightedsumgeq([1,-1], [V1,V7], V52)
weightedsumleq([1,-1], [V1,V7], V52)
weightedsumgeq([1,-1], [V0,V6], V51)
weightedsumleq([1,-1], [V0,V6], V51)
alldiff([ V45, V46, V47, V48, V49, V50])
weightedsumgeq([1,-1], [V5,V10], V50)
weightedsumleq([1,-1], [V5,V10], V50)
weightedsumgeq([1,-1], [V4,V9], V49)
weightedsumleq([1,-1], [V4,V9], V49)
weightedsumgeq([1,-1], [V3,V8], V48)
weightedsumleq([1,-1], [V3,V8], V48)
weightedsumgeq([1,-1], [V2,V7], V47)
weightedsumleq([1,-1], [V2,V7], V47)
weightedsumgeq([1,-1], [V1,V6], V46)
weightedsumleq([1,-1], [V1,V6], V46)
weightedsumgeq([1,-1], [V0,V5], V45)
weightedsumleq([1,-1], [V0,V5], V45)
alldiff([ V38, V39, V40, V41, V42, V43, V44])
weightedsumgeq([1,-1], [V6,V10], V44)
weightedsumleq([1,-1], [V6,V10], V44)
weightedsumgeq([1,-1], [V5,V9], V43)
weightedsumleq([1,-1], [V5,V9], V43)
weightedsumgeq([1,-1], [V4,V8], V42)
weightedsumleq([1,-1], [V4,V8], V42)
weightedsumgeq([1,-1], [V3,V7], V41)
weightedsumleq([1,-1], [V3,V7], V41)
weightedsumgeq([1,-1], [V2,V6], V40)
weightedsumleq([1,-1], [V2,V6], V40)
weightedsumgeq([1,-1], [V1,V5], V39)
weightedsumleq([1,-1], [V1,V5], V39)
weightedsumgeq([1,-1], [V0,V4], V38)
weightedsumleq([1,-1], [V0,V4], V38)
alldiff([ V30, V31, V32, V33, V34, V35, V36, V37])
weightedsumgeq([1,-1], [V7,V10], V37)
weightedsumleq([1,-1], [V7,V10], V37)
weightedsumgeq([1,-1], [V6,V9], V36)
weightedsumleq([1,-1], [V6,V9], V36)
weightedsumgeq([1,-1], [V5,V8], V35)
weightedsumleq([1,-1], [V5,V8], V35)
weightedsumgeq([1,-1], [V4,V7], V34)
weightedsumleq([1,-1], [V4,V7], V34)
weightedsumgeq([1,-1], [V3,V6], V33)
weightedsumleq([1,-1], [V3,V6], V33)
weightedsumgeq([1,-1], [V2,V5], V32)
weightedsumleq([1,-1], [V2,V5], V32)
weightedsumgeq([1,-1], [V1,V4], V31)
weightedsumleq([1,-1], [V1,V4], V31)
weightedsumgeq([1,-1], [V0,V3], V30)
weightedsumleq([1,-1], [V0,V3], V30)
alldiff([ V21, V22, V23, V24, V25, V26, V27, V28, V29])
weightedsumgeq([1,-1], [V8,V10], V29)
weightedsumleq([1,-1], [V8,V10], V29)
weightedsumgeq([1,-1], [V7,V9], V28)
weightedsumleq([1,-1], [V7,V9], V28)
weightedsumgeq([1,-1], [V6,V8], V27)
weightedsumleq([1,-1], [V6,V8], V27)
weightedsumgeq([1,-1], [V5,V7], V26)
weightedsumleq([1,-1], [V5,V7], V26)
weightedsumgeq([1,-1], [V4,V6], V25)
weightedsumleq([1,-1], [V4,V6], V25)
weightedsumgeq([1,-1], [V3,V5], V24)
weightedsumleq([1,-1], [V3,V5], V24)
weightedsumgeq([1,-1], [V2,V4], V23)
weightedsumleq([1,-1], [V2,V4], V23)
weightedsumgeq([1,-1], [V1,V3], V22)
weightedsumleq([1,-1], [V1,V3], V22)
weightedsumgeq([1,-1], [V0,V2], V21)
weightedsumleq([1,-1], [V0,V2], V21)
alldiff([ V11, V12, V13, V14, V15, V16, V17, V18, V19, V20])
weightedsumgeq([1,-1], [V9,V10], V20)
weightedsumleq([1,-1], [V9,V10], V20)
weightedsumgeq([1,-1], [V8,V9], V19)
weightedsumleq([1,-1], [V8,V9], V19)
weightedsumgeq([1,-1], [V7,V8], V18)
weightedsumleq([1,-1], [V7,V8], V18)
weightedsumgeq([1,-1], [V6,V7], V17)
weightedsumleq([1,-1], [V6,V7], V17)
weightedsumgeq([1,-1], [V5,V6], V16)
weightedsumleq([1,-1], [V5,V6], V16)
weightedsumgeq([1,-1], [V4,V5], V15)
weightedsumleq([1,-1], [V4,V5], V15)
weightedsumgeq([1,-1], [V3,V4], V14)
weightedsumleq([1,-1], [V3,V4], V14)
weightedsumgeq([1,-1], [V2,V3], V13)
weightedsumleq([1,-1], [V2,V3], V13)
weightedsumgeq([1,-1], [V1,V2], V12)
weightedsumleq([1,-1], [V1,V2], V12)
weightedsumgeq([1,-1], [V0,V1], V11)
weightedsumleq([1,-1], [V0,V1], V11)
alldiff([ V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10])
diseq(V11, 0)
diseq(V12, 0)
diseq(V13, 0)
diseq(V14, 0)
diseq(V15, 0)
diseq(V16, 0)
diseq(V17, 0)
diseq(V18, 0)
diseq(V19, 0)
diseq(V20, 0)
diseq(V21, 0)
diseq(V22, 0)
diseq(V23, 0)
diseq(V24, 0)
diseq(V25, 0)
diseq(V26, 0)
diseq(V27, 0)
diseq(V28, 0)
diseq(V29, 0)
diseq(V30, 0)
diseq(V31, 0)
diseq(V32, 0)
diseq(V33, 0)
diseq(V34, 0)
diseq(V35, 0)
diseq(V36, 0)
diseq(V37, 0)
diseq(V38, 0)
diseq(V39, 0)
diseq(V40, 0)
diseq(V41, 0)
diseq(V42, 0)
diseq(V43, 0)
diseq(V44, 0)
diseq(V45, 0)
diseq(V46, 0)
diseq(V47, 0)
diseq(V48, 0)
diseq(V49, 0)
diseq(V50, 0)
diseq(V51, 0)
diseq(V52, 0)
diseq(V53, 0)
diseq(V54, 0)
diseq(V55, 0)
diseq(V56, 0)
diseq(V57, 0)
diseq(V58, 0)
diseq(V59, 0)
diseq(V60, 0)
diseq(V61, 0)
diseq(V62, 0)

**EOF**
